jpype/_jarray.py

```diff
@@ -112,6 +112,8 @@ def __init__(self, *args, **kwargs):
         if values is not None:
             self.__javaarray__.setArraySlice(0, sz, values)
 
+    length = property(lambda self: self.__len__(), None)
```

Comment:
Aint that a bit obscure? Why not write it as a usual method returning self.`__len__`? I would agree on the property, if we wanted to define a setter.